### PR TITLE
Modify label to Label in the print statement

### PR DESCRIPTION
**Pull request by bkowshik**
_Thank you for this wonderful and detailed tutorial. Learnt so much!_

- Ref: https://www.tensorflow.org/tutorials/structured_data/time_series

### CODE REVIEW

**Review by 8bitmp3 — DISMISSED** [review]

**Review by 8bitmp3 — APPROVED** [review]

File: site/en/tutorials/structured_data/time_series.ipynb [hunk 1]
```diff
@@ -146,7 +147,7 @@
         "id": "R81Wx8WP4c3G"
       },
       "source": [
-        "This tutorial will just deal with **hourly predictions**, so start by sub-sampling the data from 10 minute intervals to 1h:"
+        "This tutorial will just deal with **hourly predictions**, so start by sub-sampling the data from 10-minute intervals to 1-hour ones:"
```
Comment (lamberta) [inline site/en/tutorials/structured_data/time_series.ipynb:150]:
```suggestion
        "This tutorial will just deal with **hourly predictions**, so start by sub-sampling the data from 10-minute intervals to one-hour intervals:"
```

a bit awkward to read. see if this makes sense for the context

File: site/en/tutorials/structured_data/time_series.ipynb [hunk 2]
```diff
@@ -611,13 +614,13 @@
...
         "Depending on the task and type of model you may want to generate a variety of data windows. Here are some examples:\n",
         "\n",
-        "1. For example, to make a single prediction 24h into the future, given 24h of history you might define a window like this:\n",
+        "1. For example, to make a single prediction 24 hours into the future, given 24 hours of history, you might define a window like this:\n",
         "\n",
-        "  ![One prediction 24h into the future.](images/raw_window_24h.png)\n",
+        "  ![One prediction 24 hours into the future.](images/raw_window_24h.png)\n",
         "\n",
-        "2. A model that makes a prediction 1h into the future, given 6h of history would need a window like this:\n",
+        "2. A model that makes a prediction 1 hour into the future, given 6 hours of history, would need a window like this:\n",
```
Comment (lamberta) [inline site/en/tutorials/structured_data/time_series.ipynb:621]:
```suggestion
        "2. A model that makes a prediction one hour into the future, given 6 hours of history, would need a window like this:\n",
```

Style: Use words for numbers zero through nine. Then numbers for 10 and up
https://developers.google.com/style/numbers?

Comment (lamberta) [reply site/en/tutorials/structured_data/time_series.ipynb:621]: (here and elsewhere in this PR)

**Review by lamberta — CHANGES_REQUESTED** [review]

File: site/en/tutorials/structured_data/time_series.ipynb [hunk 3]
```diff
@@ -611,13 +614,13 @@
...
+        "1. For example, to make a single prediction 24 hours into the future, given 24 hours of history, you might define a window like this:\n",
         "\n",
-        "  ![One prediction 24h into the future.](images/raw_window_24h.png)\n",
+        "  ![One prediction 24 hours into the future.](images/raw_window_24h.png)\n",
         "\n",
-        "2. A model that makes a prediction 1h into the future, given 6h of history would need a window like this:\n",
+        "2. A model that makes a prediction one hour into the future, given 6 hours of history, would need a window like this:\n",
         "\n",
-        "  ![One prediction 1h into the future.](images/raw_window_1h.png)"
+        "  ![One prediction 1 hour into the future.](images/raw_window_1h.png)"
```
Comment (8bitmp3) [inline site/en/tutorials/structured_data/time_series.ipynb:623]:
```suggestion
        "  ![One prediction one hour into the future.](images/raw_window_1h.png)"
```

File: site/en/tutorials/structured_data/time_series.ipynb [hunk 4]
```diff
@@ -1060,9 +1065,9 @@
       "source": [
         "## Single step models\n",
         "\n",
-        "The simplest model you can build on this sort of data is one that predicts a single feature's value, 1 timestep (1h) in the future based only on the current conditions.\n",
+        "The simplest model you can build on this sort of data is one that predicts a single feature's value—1 time step (1 hour) into the future based only on the current conditions.\n",
```
Comment (8bitmp3) [inline site/en/tutorials/structured_data/time_series.ipynb:1068]:
```suggestion
        "The simplest model you can build on this sort of data is one that predicts a single feature's value—1 time step (one hour) into the future based only on the current conditions.\n",
```

File: site/en/tutorials/structured_data/time_series.ipynb [hunk 5]
```diff
@@ -1060,9 +1065,9 @@
...
         "\n",
-        "So start by building models to predict the `T (degC)` value 1h into the future.\n",
+        "So start by building models to predict the `T (degC)` value 1 hour into the future.\n",
```
Comment (8bitmp3) [inline site/en/tutorials/structured_data/time_series.ipynb:1070]:
```suggestion
        "So, start by building models to predict the `T (degC)` value one hour into the future.\n",
```

File: site/en/tutorials/structured_data/time_series.ipynb [hunk 6]
```diff
@@ -1115,9 +1120,9 @@
         "\n",
         "Before building a trainable model it would be good to have a performance baseline as a point for comparison with the later more complicated models.\n",
         "\n",
-        "This first task is to predict temperature 1h in the future given the current value of all features. The current values include the current temperature. \n",
+        "This first task is to predict temperature 1 hour into the future, given the current value of all features. The current values include the current temperature. \n",
```
Comment (8bitmp3) [inline site/en/tutorials/structured_data/time_series.ipynb:1123]:
```suggestion
        "This first task is to predict temperature one hour into the future, given the current value of all features. The current values include the current temperature. \n",
```

File: site/en/tutorials/structured_data/time_series.ipynb [hunk 7]
```diff
@@ -1178,9 +1183,9 @@
...
         "\n",
-        "The `WindowGenerator` has a plot method, but the plots won't be very interesting with only a single sample. So, create a wider `WindowGenerator` that generates windows 24h of consecutive inputs and labels at a time. \n",
+        "The `WindowGenerator` has a plot method, but the plots won't be very interesting with only a single sample.\n",
         "\n",
-        "The `wide_window` doesn't change the way the model operates. The model still makes predictions 1h into the future based on a single input time step. Here the `time` axis acts like the `batch` axis: Each prediction is made independently with no interaction between time steps."
+        "So, create a wider `WindowGenerator` that generates windows 24 hours of consecutive inputs and labels at a time. The new `wide_window` variable doesn't change the way the model operates. The model still makes predictions 1 hour into the future based on a single input time step. Here, the `time` axis acts like the `batch` axis: each prediction is made independently with no interaction between time steps:"
```
Comment (8bitmp3) [inline site/en/tutorials/structured_data/time_series.ipynb:1188]:
```suggestion
        "So, create a wider `WindowGenerator` that generates windows 24 hours of consecutive inputs and labels at a time. The new `wide_window` variable doesn't change the way the model operates. The model still makes predictions one hour into the future based on a single input time step. Here, the `time` axis acts like the `batch` axis: each prediction is made independently with no interaction between time steps:"
```

File: site/en/tutorials/structured_data/time_series.ipynb [hunk 8]
```diff
@@ -1227,7 +1232,7 @@
         "id": "SKqQHX1K0JW-"
       },
       "source": [
-        "Plotting the baseline model's predictions you can see that it is simply the labels, shifted right by 1h."
+        "By plotting the baseline model's predictions, notice that it is simply the labels shifted right by 1 hour:"
```
Comment (8bitmp3) [inline site/en/tutorials/structured_data/time_series.ipynb:1235]:
```suggestion
        "By plotting the baseline model's predictions, notice that it is simply the labels shifted right by one hour:"
```

File: site/en/tutorials/structured_data/time_series.ipynb [hunk 9]
```diff
@@ -2264,7 +2268,7 @@
       "source": [
         "## Multi-step models\n",
         "\n",
-        "Both the single-output and multiple-output models in the previous sections made **single time step predictions**, 1h into the future.\n",
+        "Both the single-output and multiple-output models in the previous sections made **single time step predictions**, 1 hour into the future.\n",
```
Comment (8bitmp3) [inline site/en/tutorials/structured_data/time_series.ipynb:2271]:
```suggestion
        "Both the single-output and multiple-output models in the previous sections made **single time step predictions**, one hour into the future.\n",
```

File: site/en/tutorials/structured_data/time_series.ipynb [hunk 10]
```diff
@@ -611,13 +614,13 @@
...
         "Depending on the task and type of model you may want to generate a variety of data windows. Here are some examples:\n",
         "\n",
-        "1. For example, to make a single prediction 24h into the future, given 24h of history you might define a window like this:\n",
+        "1. For example, to make a single prediction 24 hours into the future, given 24 hours of history, you might define a window like this:\n",
         "\n",
-        "  ![One prediction 24h into the future.](images/raw_window_24h.png)\n",
+        "  ![One prediction 24 hours into the future.](images/raw_window_24h.png)\n",
         "\n",
-        "2. A model that makes a prediction 1h into the future, given 6h of history would need a window like this:\n",
+        "2. A model that makes a prediction one hour into the future, given 6 hours of history, would need a window like this:\n",
```
Comment (8bitmp3) [inline site/en/tutorials/structured_data/time_series.ipynb:621]:
```suggestion
        "2. A model that makes a prediction one hour into the future, given six hours of history, would need a window like this:\n",
```

File: site/en/tutorials/structured_data/time_series.ipynb [hunk 11]
```diff
@@ -1482,7 +1487,7 @@
       "source": [
         "The `baseline`, `linear` and `dense` models handled each time step independently. Here the model will take multiple time steps as input to produce a single output.\n",
         "\n",
-        "Create a `WindowGenerator` that will produce batches of the 3h of inputs and, 1h of labels:"
+        "Create a `WindowGenerator` that will produce batches of 3-hour inputs and 1-hour labels:"
```
Comment (8bitmp3) [inline site/en/tutorials/structured_data/time_series.ipynb:1490]:
```suggestion
        "Create a `WindowGenerator` that will produce batches of three-hour inputs and one-hour labels:"
```

**Review by 8bitmp3 — CHANGES_REQUESTED** [review]

**Review by 8bitmp3 — APPROVED** [review]

**Review by 8bitmp3 — APPROVED** [review]

File: site/en/tutorials/structured_data/time_series.ipynb (hunk 2 above)
Comment (8bitmp3) [reply site/en/tutorials/structured_data/time_series.ipynb]: Done. (Left the plot title unchanged with `1` and `6`.)

**Pull request merged**